apps/gdalalg_raster_tile.cpp

```diff
@@ -3206,7 +3839,27 @@ bool GDALRasterTileAlgorithm::RunImpl(GDALProgressFunc pfnProgress,
                             oResourceManager.GetErrorMsg().c_str());
             }
         }
+        else
```

Comment:
this is the branch for no threads and no processes, right?

Reply:
yes, adding a code comment